docs/repositories-configure/using-submodules.md

```diff
@@ -30,11 +28,20 @@ After we enabled submodules for your organization, do the following:
 
 ## Using submodules on new repositories
 
-You must repeat steps 2 to 4 above every time you add new repositories with submodules to Codacy.
+Every time you add new repositories with submodules to Codacy, [repeat steps 2 to 4](#enable) above.
```

Comment:
This is a convoluted way of providing the instructions. :sweat_smile: (Once again, I'm aware that this were the prior instructions.)

A simpler way of presenting the information would be to have the following sections:

* Pre-requirements (asking Support to enable the feature, update the license for Self-hosted)
* Enabling submodules on a repository (instructions to enable submodules on one repository)
* Automating submodules on new repositories (this section would apply to Self-hosted only, instructions on how to have new repositories automatically use submodules)

Since so few customers use the submodules feature, I don't think it's worthwhile investing in this though. But let me know what you think! :nerd_face:

Reply:
@prcr I restructured the content following your feedback. 🤓 
I took the chance to add the missing information identified in [DOCS-364](https://codacy.atlassian.net/browse/DOCS-364), as I was already reviewing the repository procedure.

Let me know if it's clear now. 🙏